content/enterprise_influxdb/v1.9/concepts/glossary.md

```diff
@@ -211,6 +181,15 @@ Related entries: [server](#server)
 
 The local server's nanosecond timestamp.
 
+## passive node (experimental)
+
+Passive nodes are data nodes that do not own any shards. They accept write calls but do not write to their own disks, perform shard lookup and RPC calls (on non-passive nodes) and distribute writes to the data node that own shards.  In essence, they act as a kind of load balancer. 
```

Comment:
I suggest adding a little to the description to make it clear they have performance implications.
"act as a kind of load balancer and write preprocessor"

Reply:
Done